centos/8/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:8
+MAINTAINER Alexander V. Tikhonov <avtikhon@tarantool.org>
+
+# Enable extra tools
+RUN yum -y install wget yum-utils
+
+# Enable extra repositories
+RUN yum -y install epel-release
```

Comment:
It is [recommended](https://fedoraproject.org/wiki/EPEL) to enable PowerTools repository when enabling EPEL. AFAIU it is a collection of developer oriented packages. I also [found](https://centos.pkgs.org/8/centos-powertools-x86_64/libyaml-devel-0.1.7-5.el8.x86_64.rpm.html) that it contains libyaml-devel package that we'll need for building of tarantool (at least for 1.10 and 2.1, see [here](https://github.com/tarantool/doc/issues/960)).

I made a test: excluded libyaml-devel from build requirements in the tarantool RPM spec and enabled -DENABLE_BUNDLED_LIBYAML:BOOL=OFF back in the spec on tarantool master (to imitate 1.10/2.1 build) and got an error during cmake stage w/o libyaml-devel package.